### PR TITLE
modulessss

### PR DESCRIPTION
**Pull request by eliotttesnier**

### CODE REVIEW

**Review by NathanBezard — APPROVED** [review]

File: src/Modules/TimeModule.hpp
Comment (Thomaltarix) [inline src/Modules/TimeModule.hpp:1]: it's in comments

**Review by Thomaltarix — CHANGES_REQUESTED** [review]

**Review by Nolfews — APPROVED** [review]

File: src/Modules/RotatableModule.hpp
Comment (Thomaltarix) [inline src/Modules/RotatableModule.hpp:1]: It must inherite from Module

**Review by Thomaltarix — CHANGES_REQUESTED** [review]

File: src/Modules/RotatableModule.hpp [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+*/
+
+#pragma once
+
+#include <GameObject.hpp>
+
+class RotatableModule : public Module {
+    public:
+        RotatableModule(GameObject *gameObject);
+        ~RotatableModule();
```
Comment (Thomaltarix) [inline src/Modules/RotatableModule.hpp]: It must declare update()

**Review by Thomaltarix — CHANGES_REQUESTED** [review]

File: src/Modules/RotatableModule.cpp [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+** Rotatable
+*/
+
+#include <RotatableModule.hpp>
+#include <MovementModule.hpp>
+
+RotatableModule::RotatableModule(GameObject *gameObject)
+{
+    if (gameObject->hasModule<RotatableModule>())
+        throw std::runtime_error("GameObject already has a RotatableModule!");
```
Comment (Thomaltarix) [inline src/Modules/RotatableModule.cpp]: Define an error class in the Module class an then throw this type of error like this :"throw Error("msg..")"

**Review by Thomaltarix — CHANGES_REQUESTED** [review]
Something to change but ask to landry before doing it

**Review by Thomaltarix — APPROVED** [review]
Finally, it's been a long time

**Review by landryarki — APPROVED** [review]

**Pull request merged**